app/components/support_interface/audit_trail_item_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<tr class="govuk-table__row gov uk-accordion__section" data-qa="audit-trail-item">
```

Comment:
This looks like something leftover from when/if audit trails used an accordion. Not quite sure how this got here, to be honest.